public/app/features/admin/migrate-to-cloud/EmptyState/InfoPane1.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+
+import { Box, Stack } from '@grafana/ui';
+import { t, Trans } from 'app/core/internationalization';
+
+import { InfoItem } from './InfoItem';
+
+export const InfoPane1 = () => {
+  return (
+    <Box padding={2} backgroundColor="secondary">
+      <Stack gap={2} direction="column">
+        <InfoItem
+          title={t('migrate-to-cloud.what-is-cloud.title', 'What is Grafana Cloud?')}
+          linkTitle={t('migrate-to-cloud.what-is-cloud.link-title', 'Learn about cloud features')}
+          linkHref="https://grafana.com/products/cloud"
+        >
+          <Trans i18nKey="migrate-to-cloud.what-is-cloud.body">
+            {
+              "Grafana cloud is a fully managed cloud-hosted observability platform ideal for cloud native environments. It's everything you love about Grafana without the overhead of maintaining, upgrading, and supporting an installation."
+            }
```

Comment:
is this the best way to be able to include apostrophes (e.g. `It's`) in strings? 🤔

Reply:
I think you'd be able to use `&apos;` HTML char for that.